mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
```

Comment:
The more idiomatic Django way of doing this is to use [`get_or_create`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#get-or-create):

```python
code_type, created = CodeType.objects.get_or_create(
    code='govuk_slug',
    defaults={'description': 'Slug for use by GOV.UK'}
)
```

(`created` will be `True` if the object was created and `False` if it already existed; we don't need this value here so could assign it to `_` instead)